libs/ballot-interpreter/src/hmpb-rust/debug.rs

```diff
@@ -483,7 +489,7 @@ pub fn draw_scored_bubble_marks_debug_image_mut(
                     .expected_bounds
                     .left()
                     .min(scored_bubble_mark.matched_bounds.left())
-                    - option_text_width
+                    - i32ify!(option_text_width)
```

Comment:
As `text_size` now returns `(u32, u32)` instead of `(i32, i32)`, we forcibly cast to `i32` and assert that the cast succeeds.